### PR TITLE
redesign

### PR DESCRIPTION
**Pull request by hillalex**
Bit of a redesign of the layout. Some summary statistics in the form of graphs on the front page?

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+These pages contain an overview of the software output of the
+department. Software has been categorised as either [research code](/research)
+(directly authored by researchers) or [tools](/tools) (primarily authored by research 
+software engineers). There is a high degree of inter-connection between the 
+various tools and research outputs. To illustrate 
+the ways in which research and tools interact, see the graphs listed on the 
+[connections](/connections) page.
+
```
Comment (richfitz) [inline content/_index.md:16]: how does the summary graph end up on this page?

File: content/all.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
++++
+title = "All"
```
Comment (richfitz) [inline content/all.md]: is it possible to get this in tab view again? I think I preferred that

File: content/odin.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
++++
+title = "Odin"
+type = "graph"
++++
+
+The _odin_ package is a "domain specific language", hosted
+in R, for representing and compiling ordinary differential equations.
+Currently it is being used within the department for research
+on malaria, measles, HIV and flu.
```
Comment (richfitz) [inline content/odin.md]:
```suggestion
on malaria, measles, HIV, flu and COVID.
```

File: static/home.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        const numPy = repos.filter(r => r.language === "python").length;
+        const numJs = repos.filter(r => r.language === "js" || r.language === "typescript").length;
+        const numTot = repos.length;
+
+        const pieData = {
+            labels: [
+                'R',
+                'Python',
+                "Javascript",
+                'Other'
```
Comment (richfitz) [inline static/home.js:13]: What do we have in this Other category? it's quite a lot!

**Review by richfitz — CHANGES_REQUESTED** [review]
Is it possible to get the graphs and software lists within a tabset again? I think that made for natural navigation. Otherwise, this is really nice

Also: what is the sort order for the cards? I couldn't work it out at a glance

**Comment by hillalex** [comment]
> Is it possible to get the graphs and software lists within a tabset again? I think that made for natural navigation. Otherwise, this is really nice
> 
> Also: what is the sort order for the cards? I couldn't work it out at a glance

Oh yeah, I forgot to add sorting

File: content/_index.md (hunk 1 above)
Comment (hillalex) [reply content/_index.md:16]: Via the index layout: https://github.com/mrc-ide/mrc-ide.github.io/pull/3/files#diff-96762550561f76f989ced99c4a0751e94696682490ee94b2d77a66377c9619ebR12

could be moved here though

File: static/home.js (hunk 4 above)
Comment (hillalex) [reply static/home.js:13]: Screenshot from 2022-08-16 15-52-33

Comment (hillalex) [reply static/home.js:13]: so mostly shell and html

**Review by richfitz — APPROVED** [review]

**Pull request merged**